Fix bundling issue

While importing package below error is getting occurred

`Module not found: Error: Can't resolve 'worker_threads' in '/Volumes/Data/dashboard-ui/node_modules/@blotoutio/ui-kit/cjs'`

--- a/package.json
+++ b/package.json
@@ -34,7 +34,6 @@
   "devDependencies": {
     "@rollup/plugin-babel": "^5.3.0",
     "@rollup/plugin-commonjs": "^21.0.1",
-    "@rollup/plugin-node-resolve": "^13.1.2",
     "@storybook/addon-essentials": "^6.1.21",
     "@storybook/addon-links": "^6.3.6",
     "@storybook/builder-webpack5": "^6.3.8",
@@ -51,7 +50,8 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.1",
     "rollup": "^2.59.0",
-    "rollup-plugin-svg": "^2.0.0"
+    "rollup-plugin-svg": "^2.0.0",
+    "styled-components": "^5.2.1"
   },
   "publishConfig": {
     "access": "public"

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,5 +1,4 @@
 import commonjs from '@rollup/plugin-commonjs'
-import { nodeResolve } from '@rollup/plugin-node-resolve'
 
 import babel from '@rollup/plugin-babel'
 import svg from 'rollup-plugin-svg'
@@ -19,10 +18,9 @@ module.exports = () => {
     inlineDynamicImports: true,
     plugins: [
       babel({
-        presets: ['@babel/preset-react'],
+        presets: [['@babel/preset-react', { runtime: 'automatic' }]],
         babelHelpers: 'bundled',
       }),
-      nodeResolve({ preferBuiltins: false }), // or `true`
       commonjs(),
       svg(),
     ],

